housekeeper/cli/add.py

```diff
@@ -70,6 +72,13 @@ def bundle_cmd(context: click.Context, bundle_name: str, json: str):
     store.session.add(new_bundle)
     new_version.bundle: Bundle = new_bundle
     store.session.add(new_version)
+    if not exclude:
+        try:
+            include_version(context.obj[ROOT], new_version)
+        except VersionIncludedError as error:
+            LOG.warning(error.message)
```

Comment:
Im thinking this should be an error message
```suggestion
            LOG.error(error.message)
```

Reply:
Open for it either way! Now I mirrored how the include command looks.

Reply:
Changed to Log.error